spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/reactive/LoadBalancerProperties.java

```diff
@@ -41,4 +48,55 @@ public void setZone(String zone) {
 		this.zone = zone;
 	}
 
+	public HealthCheck getHealthCheck() {
+		return healthCheck;
+	}
+
+	public void setHealthCheck(HealthCheck healthCheck) {
+		this.healthCheck = healthCheck;
+	}
+
+	public static class HealthCheck {
+
+		/**
+		 * Initial delay value for the HealthCheck scheduler.
+		 */
+		private int initialDelay = 0;
```

Comment:
These can be of type `Duration` so the user can say `30s` and there's no need for a separate time unit.